### PR TITLE
Moves A&A and requests to MCV 

### PR DESCRIPTION
**Pull request by pdyba**
experiments with unitests with pytest

### CODE REVIEW

File: aioquiz/tests/test_views_user.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+mock_request = Request(b'/xxx/', {'authorization': 'asdadas'}, "", "", MagicMock)
+
+# def user_required_hacked(access_level='any_user', msg='NOT AUTHORISED', code=401):
+#     def decorator(func):
+#         @wraps(func)
+#         async def func_wrapper(self, *args, **kwargs):
+#             return await func(self, *args, **kwargs)
+#         return func_wrapper
+#     return decorator
```
Comment (tqZK) [inline aioquiz/tests/test_views_user.py]: commented code

File: aioquiz/tests/test_views_user.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+
+
+# @pytest.fixture(autouse=True)
+# def nouser(monkeypatch):
+#     monkeypatch.setattr("views.utils.user_required", user_required_hacked)
```
Comment (tqZK) [inline aioquiz/tests/test_views_user.py]: commented code

File: aioquiz/tests/test_views_user.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+
+from views.user import UserView
+from views.user import INeedHelpView
+from views.user import SeatView
+from views.user import ForgotPasswordView
+from views.user import ChangePasswordView
+from views.user import SaveGDPR
```
Comment (tqZK) [inline aioquiz/tests/test_views_user.py]: import order :P

File: aioquiz/tests/test_views_user.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+#     patched_get_by_many_field_value.assert_awaited_once()
+#
+# @patch("models.common.Users.get_all")
+# async def test_user_view_get_get_all(patched_get_all):
+#     patched_get_all.return_value = True
+#     view = UserView()
+#     resp = await view.get(mock_request, MagicMock)
+#     assert isinstance(resp, HTTPResponse)
+#     assert resp.body == b'{"msg":"NOT AUTHORISED"}'
+#     patched_get_all.assert_awaited_once()
```
Comment (tqZK) [inline aioquiz/tests/test_views_user.py]: commented code

File: aioquiz/views/utils.py [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+from models import Users
+
+_users = {}
+_users_names = {}
+
+
+def user_required(access_level='any_user', msg='NOT AUTHORISED', code=401):
+    """
+    no_user - anonymus
```
Comment (tqZK) [inline aioquiz/views/utils.py]: anonymous

File: aioquiz/views/utils.py [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+    any_user - loged user
```
Comment (tqZK) [inline aioquiz/views/utils.py]: logged

File: aioquiz/views/utils.py [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+    mentor
+    organiser
+    admin
+    :param code: int
+    :param msg: str
+    :param access_level:
+    :return: wrapped_function
+    """
+    def decorator(func):
```
Comment (tqZK) [inline aioquiz/views/utils.py]: maybe more smart name?

File: aioquiz/views/utils.py [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+        @wraps(func)
+        async def func_wrapper(cls, *args, **kwargs):
+            global _users
```
Comment (grzegorzpro) [inline aioquiz/views/utils.py]: I think you don't need this line

File: aioquiz/views/utils.py [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+            resp = json({'msg': msg}, status=code)
+            try:
+                req = args[0]
+            except IndexError:
+                print(cls)
```
Comment (grzegorzpro) [inline aioquiz/views/utils.py]: Maybe a more verbose message?

**Pull request merged**